Doc/using/cmdline.rst

```diff
@@ -918,6 +918,14 @@ conflict.
    .. versionadded:: 3.7
       See :pep:`540` for more details.
 
+.. envvar:: PYTHONHISTORY
```

Comment:
New env vars added in 3.13+ need an underscore: 

https://discuss.python.org/t/change-environment-variable-style/35180?u=hugovk

```suggestion
.. envvar:: PYTHON_HISTORY
```